InteractiveHtmlBom/ecad/schema/genericjsonpcbdata_v1.schema

```diff
@@ -0,0 +1,574 @@
+            "additionalProperties": false,
+            "properties": {
+                "_type": {
+                    "type": "string"
+                },
+                "_spec_version": {
+                    "type": "integer"
+                },
+                "pcbdata": {
+                    "$ref": "#/definitions/Pcbdata"
+                },
+                "components": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Component"
+                    }
+                }
+            },
+            "minProperties": 4,
+            "title": "GenericJSONPCBData"
+        },
+        "Component": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "attr": {
+                    "type": "string"
+                },
+                "footprint": {
+                    "type": "string"
+                },
+                "layer": {
+                    "$ref": "#/definitions/Layer"
+                },
+                "ref": {
+                    "type": "string"
+                },
+                "val": {
+                    "type": "string"
+                }
+            },
+            "minProperties": 5,
+            "title": "Component"
+        },
+        "Pcbdata": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "edges_bbox": {
+                    "$ref": "#/definitions/EdgesBbox"
+                },
+                "edges": {
+                    "$ref": "#/definitions/DrawingArray"
+                },
+                "silkscreen": {
+                    "$ref": "#/definitions/DrawingSet"
+                },
+                "fabrication": {
+                    "$ref": "#/definitions/DrawingSet"
+                },
+                "footprints": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/Footprint"
+                    }
+                },
+                "metadata": {
+                    "$ref": "#/definitions/Metadata"
+                },
+                "bom": {
+                    "$ref": "#/definitions/BOM"
+                },
+                "font_data": {
+                    "$ref": "#/definitions/FontData"
+                },
+                "tracks": { 
+                    "$ref": "#/definitions/Tracks"
+                },
+                "zones": {
+                    "$ref": "#/definitions/Zones"
+                },
+                "nets": { 
+                    "type": "array",
+                    "items": { "type": "string" }
+                }
+            },
+            "required": [
+                "edges_bbox",
+                "edges",
+                "silkscreen",
+                "fabrication",
+                "footprints",
+                "metadata",
+                "bom",
```

Comment:
not required, in fact shouldn't be in the schema

Reply:
eliminated "bom" from the schema